Server/VikingVault.Services/Exceptions/TransactionException.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace VikingVault.Services.Exceptions
+{
+    public class TransactionException:Exception
+    {
+
```

Comment:
You can create a constructor for this exception with a string as a parameter, in order to have a predefined format for your exception message

Reply:
Done.